### DISCUSSION TITLE
Resident Evil 4 Remake version 1.02 CUSA33388

### DISCUSSION THREAD

**Discussion by ssvillamor23** [GoldHEN/GoldHEN_Cheat_Repository#172] [Cheat requests]
Thanks for making the SHN version of the cheat.

I would just like to request one additional cheat if its possible which is

 Exclusive Upgrade Tickets

Can it be possible to multiple , edit or max it?

Thanks again.

**Comment by kmeps4** [comment]
Hello

Sorry we don't develop cheats on GoldHEN Organization. You will have to go to PS4Trainer Discord
